ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings

```diff
@@ -1101,3 +1101,16 @@ Puoi annullare il tuo abbonamento in ogni momento dalle impostazioni del tuo acc
 "Get notified when a news is posted" = "Ricevi una notifica quando una notizia viene pubblicata";
 "ACNew.notification.title" = "È stata pubblicata una notizia";
 "ACNew.notification.subtitle" = "È stato pubblicato un nuovo articolo sull'app o sul gioco. Vai a leggerlo nell'app!";
+
+// Summer Patch - added 2020-07-04
+"Underwater" = "Sott'acqua";
+"Movement speed:" = "Velocità movimento:";
+"Very slow" = "Molto lenta";
+"Slow" = "Lenta";
+"Very fast" = "Molto veloce";
+"Fast" = "Veloce";
+"Stationary" = "Statica";
+
+// Added 2020-07-07
+"Clothing Other" = "Altri indumenti";
```

Comment:
Actually the english string is really bad 😆 The correct one would be "diving suit" - so you may can replace your translation with the Italian word for diving suit?

Reply:
Indeed I was a little confused about it 😅. In Italian diving suit is "Muta subacquea"
Can I change the file without a new pull request?

Reply:
Seems you got it 😊 adding ne commits to a branch that has an active pull request will add these commits automatically to the pull request.